test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -1322,6 +1323,14 @@
           }
         }
       },
+      "Microsoft.Security.Utilities/1.1.0": {
```

Comment:
@fabiocav / @brettsam to help review these deps changes

Reply:
I just updated this deps file to properly reflect the current Microsoft.Security.Utilities dependency and the corresponding test passes. So just need to validate with everyone we are comfortable with picking up this Microsoft NuGet dependency.

Reply:
@mathewc, FYI, I advertently cleared your previous approval, oops.